src/client/src/Stream.c

```diff
@@ -3245,3 +3246,46 @@ STATUS resetStream(PKinesisVideoStream pKinesisVideoStream)
     return retStatus;
 }
 
+VOID logStreamInfo(PStreamInfo pStreamInfo) {
+    PCHAR streamingType;
+
+    if (pStreamInfo == NULL) {
+        return;
+    }
+
+    switch (pStreamInfo->streamCaps.streamingType) {
+        case STREAMING_TYPE_REALTIME:
+            streamingType = (PCHAR) "REALTIME";
+            break;
+        case STREAMING_TYPE_OFFLINE:
+            streamingType = (PCHAR) "OFFLINE";
+            break;
+        case STREAMING_TYPE_NEAR_REALTIME:
+            streamingType = (PCHAR) "NEAR_REALTIME";
+            break;
+    }
+
+
+    DLOGD("Kinesis Video Stream Info:\n\t"
+                  "Stream name: %s\n\t"
+                  "Streaming type: %s\n\t"
+                  "Max latency (100ns): %" PRIu64 "\n\t"
+                  "Fragment duration (100ns): %" PRIu64 "\n\t"
+                  "Key frame fragmentation: %s\n\t"
+                  "Absolute frame timecode: %s\n\t"
+                  "Buffer duration (100ns): %" PRIu64 "\n\t"
+                  "Replay duration (100ns): %" PRIu64 "\n\t"
+                  "Connection Staleness duration (100ns): %" PRIu64 "\n\t"
+                  "Frame order mode: %u",
+          pStreamInfo->name,
+          streamingType,
```

Comment:
Would be nice to stringify this. Have a small macro that returns a pre-canned string for Realtime/Offline